src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/UniqueAltReadCount.java

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypeBuilder;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFFormatHeaderLine;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.utils.genotyper.ReadLikelihoods;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * This annotation counts, at each variant locus, the number of ALT reads with unique start positions and fragment lengths.
+ * i.e. multiple reads with the same start position and fragment length are grouped and counted only once as they are
+ * likely duplicates. We have seen that the majority of false positives in low allele fraction, cell-free DNA samples
+ * have the profile where the evidence for alternate allele comes solely from one, two, or three sets of apparent PCR-duplicates.
+ * Reads in such a set have the same read-start and mate-end position i.e. they come from the same original insert.
+ * Normally, they are marked as duplicates by Picard MarkDuplicates. But when we use unique molecular identifiers (UMIs),
+ * these apparent PCR-duplicate reads may get different UMIs, and hence these reads are regarded independent evidence.
+ *
+ * Although these reads have different UMIs, we suspect that they really are PCR-duplicates, for two reasons:
+ * 1) these sites are false positives, and
+ * 2) with hybrid selection, it's highly unlikely that we get multiple fragments with identical start and end positions.
+ *
+ * We don't understand why these apparent PCR-duplicates get different UMIs; investigation is underway.
+ *
+ * We filter the variant if the count is lower than a user-specified threshold.
```

Comment:
I don't think this is true, is it?  We want to filter the variant if the ratio of the count of duplicate alt reads to total count of alt reads, right?